code/frameworks/nextjs/template/stories/Image.stories.jsx

```diff
@@ -48,3 +48,30 @@ export const Sized = {
     ],
   },
 };
+
+export const Lazy = {
+  args: {
+    src: 'https://storybook.js.org/images/placeholders/50x50.png',
+    width: 50,
+    height: 50,
+  },
+  decorators: [
+    (Story) => (
+      <>
+        <div style={{ height: '200vh' }} />
```

Comment:
To make sure lazy image's don't get loaded I added a `200vh` tall div. The point where a lazy loaded image gets loaded is different for every browser, but `200vh` works for both Chrome and Safari.